src/sql/workbench/contrib/query/browser/queryInputFactory.ts

```diff
@@ -28,7 +28,7 @@ const editorInputFactoryRegistry = Registry.as<IEditorInputFactoryRegistry>(Edit
 
 export class QueryEditorLanguageAssociation implements ILanguageAssociation {
 	static readonly isDefault = true;
-	static readonly languages = ['sql'];
+	static readonly languages = ['sql', 'kusto'];			//TODOKusto Add language id here for new languages supported in query editor. Make it easier if possible.
```

Comment:
Get rid of the extra spacing here. And please include link to tracking item - although I understand the comment. You're already adding the language id right? So shouldn't the work item just be to make this easier to contribute to?